endpoints/put_test.go

```diff
@@ -989,30 +1034,17 @@ func doMockGet(t *testing.T, router *httprouter.Router, id string) *httptest.Res
 	return requestRecorder
 }
 
-func doMockPut(t *testing.T, router *httprouter.Router, content string) (string, *httptest.ResponseRecorder) {
-	var parseMockUUID = func(t *testing.T, putResponse string) string {
-		var parsed PutResponse
-		err := json.Unmarshal([]byte(putResponse), &parsed)
-		if err != nil {
-			t.Errorf("Response from POST doesn't conform to the expected format: %v", putResponse)
-		}
-		return parsed.Responses[0].UUID
-	}
-
+func doMockPut(t *testing.T, router *httprouter.Router, content string) *httptest.ResponseRecorder {
 	rr := httptest.NewRecorder()
 
 	request, err := http.NewRequest("POST", "/cache", strings.NewReader(content))
 	if err != nil {
 		t.Fatalf("Failed to create a POST request: %v", err)
-		return "", rr
+		return rr
 	}
-
 	router.ServeHTTP(rr, request)
-	uuid := ""
-	if rr.Code == http.StatusOK {
-		uuid = parseMockUUID(t, rr.Body.String())
-	}
-	return uuid, rr
+
+	return rr
```

Comment:
Why did you remove the uuids and response parsing? That seems like worthwhile information. We were not using it before?

Reply:
The first parameter was being ignored almost everywhere. In the one test that made use of it I simply extracted the UUID from the recorder body in the test itself.